test/test-pages/buzzfeed-1/expected-metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Student Dies After Diet Pills She Bought Online \"Burned Her Up From Within\"",
-  "byline": "Mark Di Stefano",
-  "excerpt": "An inquest into Eloise Parry's death has been adjourned until July...",
+  "byline": null,
```

Comment:
The source has ` <meta property="author" content="Mark Di Stefano"/>`. Can we continue matching that if we have nothing else?

Reply:
This is a mistake by the document author. The property has no prefix and the element doesn’t have a different namespace. This is an unknown property. The website have removed this tag today and only supply the author through JSON-LD. Adding support for JSON-LD would resolve this and detect authors for many test cases that lack them.